test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -71,7 +72,7 @@ import {
   BO_GOV_ID_URL,
   MO_IND_ID_URL,
   MO_CORP_ID_URL,
-  TRANSACTION_ID, PUBLIC_REGISTER_NAME, PUBLIC_REGISTER_JURISDICTION, REGISTRATION_NUMBER,
+  TRANSACTION_ID, PUBLIC_REGISTER_NAME, PUBLIC_REGISTER_JURISDICTION, REGISTRATION_NUMBER, TRUST
```

Comment:
These should be on separate lines. Appreciate this line was already wrong, but wouldn't perpetuate this.

Reply:
Updated

Reply:
Thanks.